src/renderer/mod.rs

```diff
@@ -60,7 +60,16 @@ impl Renderer {
         }
     }
 
+    #[cfg(not(feature = "testing-colors"))]
```

Comment:
I'd recommend having one public facing method
- Easier to avoid signature drift
- You only get the docs for one of them (and docs.rs matters the most) so you should have the documentation the same and doing that is the easiest when you have one public signature